client/tests/webdriver/pages/report/showReport.page.js

```diff
@@ -81,10 +81,11 @@ class ShowReport extends Page {
   }
 
   waitForShowReportToLoad() {
-    if (!this.reportStatus.isDisplayed()) {
-      this.reportStatus.waitForExist()
-      this.reportStatus.waitForDisplayed()
-    }
+    browser.waitUntil(() =>
+      /^.*\/reports\/[a-z0-9-]{36}/.test(browser.getUrl())
```

Comment:
This also matches the Edit page URL. Perhaps hook the regex at the end with a `$`?

Reply:
It is supposed to match the Edit page URL, as well. On `keycloak-integration` branch I noticed that there is an intermediate URL change when navigating between pages. This change aims to handle that gracefully.